tests/Codeception/_support/Step/Acceptance/Administrator/Field.php

```diff
@@ -118,6 +118,7 @@ public function deleteField($title, $message)
 		$I->checkAllResults();
 		$I->wait(2);
 		$I->clickToolbarButton('Empty trash');
-		$I->assertSuccessMessage($message);
+		$I->wait(2);
+		$I->acceptPopup();
```

Comment:
```suggestion
		$I->acceptPopup();
		$I->assertSuccessMessage($message);
```
@brianteeman Could you apply this change so the old behaviour of this test is restored? Otherwise, without that line, the `$message` parameter would be useless here: https://github.com/joomla/joomla-cms/blob/36524df7974444596b9a4c16645db4936ca0e6d6/tests/Codeception/_support/Step/Acceptance/Administrator/Field.php#L102

I agree with you that we should check the same method at other places and if necessary fix them, too.

Reply:
P.S.: It will not need new human tests after this change will be applied.

Reply:
i will commit the suggestion as yu asked nicely. i still believe it to be wrong to do it in this pr but ...

Reply:
Thanks. Much appreciated. I agree with you at least to a large part but wanted to avoid arguments with other maintainers 😉 .